src/DIRAC/Core/Tornado/scripts/tornado_start_web.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import os
+import sys
+import tornado
+import pprint
+
+from DIRAC.Core.Utilities.DIRACScript import DIRACScript
+
+
+@DIRACScript()
```

Comment:
This needs to be added to the `setup.cfg`

Reply:
added